EIPS/eip-2681.md

```diff
@@ -26,7 +26,7 @@ Lastly, this facilitates a minor optimisation in clients, because the nonce no l
 Introduce two new restrictions retroactively from genesis:
 
 1. Consider any transaction invalid, where the nonce exceeds or equals to `2^64-1`.
-2. The `CREATE` and `CREATE2` instructions to abort with an exceptional halt, where the account nonce is `2^64-1`.
+2. The CREATE and CREATE2 instructions execution ends with the result `0` pushed on stack, where the account nonce is `2^64-1`. Gas for initcode execution is not deducted in this case.
```

Comment:
```suggestion
2. The `CREATE` and `CREATE2` instruction's execution ends with the result `0` pushed on stack, where the account nonce is `2^64-1`. Gas for initcode execution is not deducted in this case.
```

Reply:
Heh, I initially used `instructions'` in my suggestion by my confidence on when the right time to do that is incredibly low so I went with the "safer" `instruction's` and hoped no one would notice/correct me.  😬 

I *think* `instructions'` is correct... if you are confident then lets go with that!

Reply:
```suggestion
2. The `CREATE` and `CREATE2` instructions' execution ends with the result `0` pushed on stack, where the account nonce is `2^64-1`. Gas for initcode execution is not deducted in this case.
```

Reply:
Changed.